modules/monitoring/manifests/checks.pp

```diff
@@ -45,6 +45,10 @@
   include monitoring::checks::grafana_dashboards
   include monitoring::checks::cdn_logs
 
+  if $::aws_migration {
+    include govuk::apps::publisher::unprocessed_emails_count_check
+  }
+
   if $::aws_migration {
     include govuk::apps::email_alert_api::checks
   }
```

Comment:
I think we can combine these two into one if statement:

```puppet
if $::aws_migration {
  include govuk::apps::publisher::unprocessed_emails_count_check
  include govuk::apps::email_alert_api::checks
}
```

Reply:
Ah, yes, of course! 😅
I missed that for some reason! 🤦